packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -306,7 +306,7 @@ const isSticky = (post, terms) => {
   }
 }
 
-const PostsItem2 = ({
+const PostsItem2: React.FC<PostsItem2Props> = ({
```

Comment:
I pretty strongly prefer us to import `FC` from `react` directly here. So just replace this with `FC<PostsItem2Props>` and then import FC in the import React line.